src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -453,16 +478,20 @@ def main():
     root_dir = getattr(args, loader_common_names.ROOT_DIR_ARG_NAME)
     token_filepath = args.token_filepath
     upload_file_ext = args.upload_file_ext
-    workers = args.workers
+    batch_size = args.batch_size
```

Comment:
If batch size is really just for tracking how much work has been done it seems like it doesn't really need to be a parameter. We could just choose a reasonable value

Reply:
It will also control how frequent the progress message gets print out. I would like to keep this param.

Reply:
Ok. I think we should do something better in the future, like having assembly util write its progress to a file, though. Could you make a github issue for that?

Reply:
https://github.com/kbase/collections/issues/575

Reply:
I was thinking of a ticket in assemblyutil, since the work will have to be done there, although a tracking ticket in collections as well isn't a bad idea.

Reply:
https://github.com/kbaseapps/AssemblyUtil/issues/92

Reply:
:+1: